web/src/search_suggestion.ts

```diff
@@ -134,12 +134,9 @@ const incompatible_patterns: Partial<Record<NarrowTerm["operator"], TermPattern[
     "has:reaction": [{operator: "has", operand: "reaction"}],
 };
 
+// TODO: We have stripped suggestion of all other attributes, we should now
+// replace it with simple string in other places.
```

Comment:
We should probably do this as a followup or at least open an issue for it once this PR is merged.